protocols/v2/noise-sv2/src/cipher_state.rs

```diff
@@ -80,6 +150,17 @@ where
     }
 }
 
+// An enum representing a generic cipher state.
```

Comment:
Perhaps rephrase or add context to the `GenericCipher` enum comments based on the suggestions for the top level mod comments in this module, given that the top level module doc comments are private and will not render to crates.io, but the `GenericCipher` enum  doc comments are public and will render.